### PR TITLE
Update CONTRIBUTING.md to reflect PR merge process

### PR DESCRIPTION
**Pull request by yifeif**

### CODE REVIEW

**Review by gunan — DISMISSED** [review]

**Comment by yifeif** [comment]
Thanks @gunan!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ Follow either of the two links above to access the appropriate CLA and instructi
 If you have improvements to TensorFlow, send us your pull requests! For those
 just getting started, Github has a [howto](https://help.github.com/articles/using-pull-requests/).
 
-TensorFlow team members will be assigned to review your pull requests. Once the pull requests are approved and pass continuous integration checks, we will merge the pull requests.
+TensorFlow team members will be assigned to review your pull requests. Once the pull requests are approved and pass continuous integration checks, a TensorFlow team member will apply `ready to pull` label to your change. This means we are working on getting your pull request submitted to our internal repository. After the change has been submitted internally, your pull request will be merged automatically on GitHub.
+
 For some pull requests, we will apply the patch for each pull request to our internal version control system first, and export the change out as a new commit later, at which point the original pull request will be closed. The commits in the pull request will be squashed into a single commit with the pull request creator as the author. These pull requests will be labeled as pending merge internally.
```
Comment (Androbin) [inline CONTRIBUTING.md]: What about this next paragraph? Isn't part of it redundant now?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @yifeif: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by drpngx** [comment]
@yifeif @Androbin just checking that this is ready to go?

File: CONTRIBUTING.md (hunk 1 above)
Comment (yifeif) [reply CONTRIBUTING.md]: Done. Thanks!

**Comment by yifeif** [comment]
Thanks @drpngx. It should be ready now.

**Pull request merged**